tests/custom_lookups/tests.py

```diff
@@ -420,94 +408,97 @@ def test_datetime_output_field(self):
 
 class YearLteTests(TestCase):
     def setUp(self):
-        models.DateField.register_lookup(YearTransform)
```

Comment:
This existing approach here looks fine to me. I don't see an advantage to repeating the same logic in each test when it could be done once in setUp/tearDown?